Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -2540,6 +2560,306 @@ def pin_message():
     except SlackApiError as slack_error:
         return_error(f"{slack_error}")
 
+def list_channels():
```

Comment:
Please add a unit test for your new command.
You can use the following guide - https://xsoar.pan.dev/docs/integrations/unit-testing as well as examples of unit tests in the HelloWorld integration and other packs from the content repository.

Reply:
Done